source/includes/stackpath/_certificates.md

```diff
@@ -1,5 +1,130 @@
 ## Certificates
 
+Add Secure Socket Layer (SSL) features to a site.
+
+<!-------------------- LIST CERTIFICATES -------------------->
+
+### List certificates
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/certificates?siteId=0a57855b-26d8-4e8f-8b77-429997c7c5fb"
+```
+> The above command returns a JSON structured like this:
+```json
+{
+  "data": [
+    {
+      "id": "2a12237d-c388-4c19-bff1-b019b837e8f1",
+      "issuer": "",
+      "caBundle": "",
+      "status": "failed",
+      "expiresAt": "2021-04-01T18:11:11.903458Z",
+      "commonName": "helloworld.com",
+      "createdAt": "2021-04-01T18:11:11.904285Z",
+      "isManaged": true,
+      "trusted": false,
+      "updatedAt": "2021-04-04T18:30:32.021842Z"
```

Comment:
If it's possible to have some realistic looking values for `issuer`, `caBundle` and `commonName` then that would be good.